Adding logs to the repo (#91)
Tenderly/hardhat-tenderly: packages/tenderly-hardhat/README.md

};
```

> If you are having trouble with the plugin and want to contact support, you can rum the deploy script with the following ```--verbose``` flag as so:

Reviewer: ```suggestion
If you are having trouble with the plugin and want to contact support, you can run the deploy script with the following ```--verbose``` flag as so:
```

```bash
npx hardhat run scripts/{your_deploy_script_here.js} --network {network_name} --verbose > tenderly.log 2>&1
```

> This will create a ```tenderly.log``` file that you can send to our customer support.

Reviewer: ```suggestion
This will create a ```tenderly.log``` file that you can send to our customer support engineers for investigation.
```
